backend/wmg/pipeline/summary_cubes/calculate_markers.py

```diff
@@ -101,6 +103,17 @@ def _query_tiledb_context_memoized(
     query = q.expression_summary_fmg(criteria)
     cell_counts_query = q.cell_counts(criteria)
 
+    healthy_filter_es = query["disease_ontology_term_id"].astype("str") == NORMAL_CELL_DISEASE_ONTOLOGY_TERM_ID
+    query = query[healthy_filter_es]
+
+    healthy_filter_cc = (
+        cell_counts_query["disease_ontology_term_id"].astype("str") == NORMAL_CELL_DISEASE_ONTOLOGY_TERM_ID
+    )
+    cell_counts_query = cell_counts_query[healthy_filter_cc]
+
+    if query.shape[0] == 0 or cell_counts_query.shape[0] == 0:
```

Comment:
If either of the filtered dataframes (expression summary or cell counts) are empty, raise an error and move on to the next cell type.